workday/13.0/modules/ROOT/pages/workday-connector-upgrade-migrate.adoc

```diff
@@ -8,11 +8,16 @@ Upgrade Anypoint Connector for Workday (Workday Connector) to version 13.x.x.
 [%header,cols="50a,50a"]
 |===
 |From Version | To Version
+|12.1.0 |13.x.x
```

Comment:
Is the upgrade path:
11.x.x > 13.x.x  
?
Or is it really 11.1.x ?
Verify upgrade path.

Reply:
This is upgrade from latest version which is 12.1.0 to 13.0.0

Reply:
But can you upgrade from earlier versions to 13.0?

Reply:
Theoretically yes. You can upgrade from any version to any version